pandas/tests/scalar/timestamp/test_constructors.py

```diff
@@ -34,6 +34,24 @@ def test_construct_from_string_invalid_raises(self):
         with pytest.raises(ValueError, match="gives an invalid tzoffset"):
             Timestamp("200622-12-31")
 
+    def test_constructor_str_infer_reso(self):
+        # non-iso8601 path
+
+        # _parse_delimited_date path
+        ts = Timestamp("2023/01/30")
```

Comment:
this hits the iso path, you'll need
```
ts = Timestamp("01/30/2023")
```

also, would prefer to parametrise these - you can put `_parse_delimited_date path` (and the others) as `id` for the different cases

Reply:
updated to use the suggested string.  didnt parametrize - on the margin id like to slow the growth of the pytest overhead

Reply:
just out of interest, why?

Reply:
pytest collects something like 200k tests and last time i checked that added up to more than 1GB before we even start executing any tests.  Similarly autouse fixtures get called for each test which means that much more overhead at runtime (i haven't measured this so assume it is a small-but-adds-up amount).

Doing a little bit more inside the tests and a little bit less outside can trim that footprint and overhead.  Adding a couple cases here isn't a big deal, but on the margin i try to get in the habit of leaning against parametrization when it isn't a compelling use case.